data-processing/entities/sentences/extractor.py

```diff
@@ -8,7 +8,19 @@
 from common.types import SerializableEntity
 # These are 'reserved characters' by the pysbd module and can potentially
 # cause issues if they are present in a string. This list was compiled from the
-# psybd source code as of 3/23/20
+# psybd source code as of 3/23/20. locations:
+# ∯: https://github.com/nipunsadvilkar/pySBD/blob/npn-carriage-return-fix/pysbd/abbreviation_replacer.py, https://github.com/nipunsadvilkar/pySBD/blob/npn-carriage-return-fix/pysbd/lists_item_replacer.py
```

Comment:
@zkirby awesome!! can you make these permalinks, anchored to the specific version and line, e.g., https://github.com/nipunsadvilkar/pySBD/blob/995af9e3c3d324aa053cb06f79c0038c882f633d/pysbd/lists_item_replacer.py#L21 ? To do this, you find the line on GitHub, click on the line number, click on the ellipses that appear, and then click "Copy permalink":

![image](https://user-images.githubusercontent.com/2358524/77375469-cdfa2c00-6d2a-11ea-8bdd-f4ae466afa33.png)

Reply:
Oh dang hahaha, how important is it to do this? I'm not sure it will be super helpful bc the files themselves are quite short and the symbols are scattered many times throughout the files.

Reply:
not that important! let's skip it and merge this work of art into the code base.